package proto_ff;

import "yd_fieldoptions.proto";


message E_OccupationOccupation
{
	optional int64 m_id = 1[(yd_fieldoptions.field_cname) = "转职职业ID"];
	optional int32 m_professionID = 2[(yd_fieldoptions.field_cname) = "主职业ID"];
	optional int32 m_grade = 3[(yd_fieldoptions.field_cname) = "职业等级"];
	optional string m_equip = 4[(yd_fieldoptions.field_cname) = "展示武器模型", (yd_fieldoptions.field_bufsize) = 64];
	optional string m_modelID = 5[(yd_fieldoptions.field_cname) = "展示衣服模型", (yd_fieldoptions.field_bufsize) = 64];
	optional string m_activeDesc = 6[(yd_fieldoptions.field_cname) = "激活描述", (yd_fieldoptions.field_bufsize) = 128];
	optional int32 m_level = 7[(yd_fieldoptions.field_cname) = "等级要求"];
	optional int64 m_skill_id = 8[(yd_fieldoptions.field_cname) = "增加技能ID"];
	optional string m_itemIDs = 9[(yd_fieldoptions.field_cname) = "转职道具奖励奖励", (yd_fieldoptions.field_bufsize) = 64];
}


message Sheet_OccupationOccupation
{
	repeated E_OccupationOccupation E_OccupationOccupation_List = 1[(yd_fieldoptions.field_arysize)=16];
}

message E_OccupationStage
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 m_grade = 2[(yd_fieldoptions.field_cname) = "职业等级"];
	optional string m_stgae = 3[(yd_fieldoptions.field_cname) = "阶段", (yd_fieldoptions.field_bufsize) = 64];
	optional int32 m_lv = 4[(yd_fieldoptions.field_cname) = "要求等级"];
	optional int32 m_stgaeAttShow = 5[(yd_fieldoptions.field_cname) = "属性是否展示用"];
	optional int32 m_stgaeAtt = 6[(yd_fieldoptions.field_cname) = "阶段属性"];
	optional int32 m_taskId = 7[(yd_fieldoptions.field_cname) = "阶段任务ID"];
	optional int32 m_battle = 8[(yd_fieldoptions.field_cname) = "推荐战力"];
	optional int32 m_skillID = 9[(yd_fieldoptions.field_cname) = "激活技能"];
	optional string m_lightSoul = 10[(yd_fieldoptions.field_cname) = "点亮龙魂", (yd_fieldoptions.field_bufsize) = 64];
	optional int32 m_soulTpye = 11[(yd_fieldoptions.field_cname) = "龙魂类型"];
	optional int32 m_itemID = 12[(yd_fieldoptions.field_cname) = "点亮的道具"];
}


message Sheet_OccupationStage
{
	repeated E_OccupationStage E_OccupationStage_List = 1[(yd_fieldoptions.field_arysize)=32];
}

message E_OccupationAttributeAttributeDesc
{
	optional int32 m_value = 1[(yd_fieldoptions.field_cname) = "数量"];
	optional int32 m_item = 2[(yd_fieldoptions.field_cname) = "ID"];
}

message E_OccupationAttribute
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "属性id"];
	repeated E_OccupationAttributeAttributeDesc m_attribute = 2[(yd_fieldoptions.field_cname) = "属性道具", (yd_fieldoptions.field_arysize) = 6];
}


message Sheet_OccupationAttribute
{
	repeated E_OccupationAttribute E_OccupationAttribute_List = 1[(yd_fieldoptions.field_arysize)=64];
}

message E_OccupationSoulAttributeDesc
{
	optional int32 m_value = 1[(yd_fieldoptions.field_cname) = "值"];
	optional int32 m_type = 2[(yd_fieldoptions.field_cname) = "ID"];
}

message E_OccupationSoul
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "属性id"];
	optional int32 m_soulTpye = 2[(yd_fieldoptions.field_cname) = "属性类型"];
	optional int32 m_position = 3[(yd_fieldoptions.field_cname) = "位置"];
	repeated E_OccupationSoulAttributeDesc m_attribute = 4[(yd_fieldoptions.field_cname) = "基础属性", (yd_fieldoptions.field_arysize) = 6];
}


message Sheet_OccupationSoul
{
	repeated E_OccupationSoul E_OccupationSoul_List = 1[(yd_fieldoptions.field_arysize)=64];
}
